src/Umbraco.Core/Services/LocalizedTextServiceFileSources.cs

```diff
@@ -246,16 +258,16 @@ private void MergeSupplementaryFiles(CultureInfo culture, XDocument xMasterDoc)
 
             foreach (LocalizedTextServiceSupplementaryFileSource supplementaryFile in found)
             {
-                using (FileStream fs = supplementaryFile.File.OpenRead())
+                using (var stream = supplementaryFile.FileInfo.CreateReadStream())
```

Comment:
```suggestion
                using (Stream? stream = supplementaryFile.FileInfo.CreateReadStream())
```

We should use explicit types here 😄